prepare-chroot-base

```diff
@@ -46,40 +36,37 @@ else
     YUM_BOOTSTRAP_CONF=yum-bootstrap-${DISTRIBUTION}.conf
 fi
 
-YUM="yum --downloaddir=${DOWNLOADDIR} --downloadonly install"
+YUM=(yum "--downloaddir=$DOWNLOADDIR" --downloadonly install)
```

Comment:
When you convert `$YUM` to an array, please also update all usage of the variable. I believe this is what caused the CI failure.